.travis.yml

```diff
@@ -3,7 +3,7 @@ python:
     - "3.5"
     - "3.4"
 before_install:
-    - git clone https://github.com/ONSdigital/sdx-common.git
+    - git clone --branch 0.7.0 https://github.com/ONSdigital/sdx-common.git
```

Comment:
I'm not keen on the version of sdx-common being in 3 places now. 

We could rework the travis script to at call the make file, that would remove the need to have this line twice at least.